src/FunctionLoader.cs

```diff
@@ -13,6 +17,10 @@ internal class FunctionLoader
     {
         private readonly MapField<string, AzFunctionInfo> _loadedFunctions = new MapField<string, AzFunctionInfo>();
 
+        internal static string FunctionAppRootLocation { get; set; }
+        internal static string FunctionAppProfileLocation { get; set; } = null;
+        internal static string FunctionAppModulesLocation { get; set; } = null;
```

Comment:
Can we renamed them to use `Path` suffix instead of `Location`? They are long enough ... :)

Reply:
changed